lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -58,6 +61,23 @@ class LighthouseViewerReport {
     if (shareButton) {
       shareButton.addEventListener('click', this.onShare);
     }
+
+    const copyButton = document.querySelector('.js-copy');
+    if (copyButton) {
+      copyButton.addEventListener('click', this.onCopy);
+
+      document.addEventListener('copy', e => {
```

Comment:
move this to it's own method?

also it's weird that there's `onCopy` which means that the copy button was clicked and this function, which is what would often be called `onCopy` :)